packages/vscode-extension/src/webview/components/RenderOutlinesOverlay.css

```diff
@@ -0,0 +1,16 @@
+.render-outlines-overlay {
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  pointer-events: none;
+  z-index: 2147483646;
```

Comment:
would rather not have it here. Last time we had z-index it'd show stuff over modals etc. Can this be avoided?

Reply:
This was taken directly from `react-scan`. It's probably fine without it, since we draw the canvas over the app preview (and we don't need to make sure we draw over the entire window, like the "real" `react-scan` does).